### PR TITLE
Issue 2 ghostscript

### PR DESCRIPTION
**Pull request by codecounselor**

### CODE REVIEW

File: lib/powerpoint.js [hunk 1]
```diff
@@ -61,81 +72,88 @@ class Powerpoint extends EventEmitter2 {
...
+   * @param outputDir the directory where png and pptx files are generated
+   * @private
+   */
   _aggregateSlides (images, pptxOutput, done, outputDir) {
     this._createSlides(images, pptxOutput, (slideErr, output) => {
-      pdfLogger('Finished rendering all slides')
       done(slideErr, output)
-      rmdir(path.resolve(outputDir, 'img'),
-        (e) => e && errorLogger('Could not delete working directory:', e))
+      this.clean && rmdir(path.resolve(outputDir, 'img'), (e) => e && errorLogger('Could not delete working directory:', e))
```
Comment (mattsgarlata) [inline lib/powerpoint.js]: This line is kind of hard to follow because it's so long.  I think it would be better split onto multiple lines.

Comment (codecounselor) [inline lib/powerpoint.js]: good point, I've changed this.
